app/prometheus/entrypoint.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 /bin/sed -i 's/{GRAFANA_CLOUD_TOKEN}/'$GRAFANA_CLOUD_TOKEN'/' /etc/prometheus/prometheus.yml
```

Comment:
Should this also needs curly braces?

Reply:
it should not have curly braces, the PROMETHEUS_ENVIRONMENT one is wrong

Reply:
(fixed in last commit)